src/main/kotlin/no/nav/familie/ef/sak/ekstern/EksternVedtakService.kt

```diff
@@ -32,21 +31,20 @@ class EksternVedtakService(
     }
 
     private fun tilFagsystemVedtak(behandling: Behandling): FagsystemVedtak {
-        val (resultat, vedtakType) = utledReultatOgVedtakType(behandling)
+        val (resultat, fagsystemType) = utledReultatOgFagsystemType(behandling)
 
         return FagsystemVedtak(
             eksternBehandlingId = behandling.eksternId.id.toString(),
             behandlingstype = behandling.type.visningsnavn,
             resultat = resultat,
             vedtakstidspunkt = behandling.vedtakstidspunkt
                 ?: error("Mangler vedtakstidspunkt for behandling=${behandling.id}"),
-            fagsystemType = FagsystemType.ORDNIÆR,
-            vedtakType = vedtakType
+            fagsystemType = fagsystemType,
```

Comment:
Skal VedtakType fjernes? Trodde den nyligen ble lagt till? 
I PR for tilbake ble den lagt tl nå?

Reply:
Vi har skriveleif i "fagsystemtype" (ORDNIÆR) - så da må vi migrere og fandens oldemor for å støtte sanksjon og rename feltet. 

Tenkte at renaming av felt skulle være "nok", men det funka dårlig 😂

Reply:
Det var først tenkt at VedtakTypen skulle erstatte FagsystemType, men det viste seg å bli litt mer jobb enn først antatt å få til dette. I familie-klage brukes FagsystemType i `PåklagetVedtakDetaljer` som er et database-objekt, noe som fører til litt for mye knot hvis vi skal bytte ut FagsystemType med VedtakType